c++/src/capnp/message.h

```diff
@@ -414,6 +421,21 @@ class FlatMessageBuilder: public MessageBuilder {
   bool allocated;
 };
 
+template <typename MR>
```

Comment:
Does this really need to be a template? Couldn't it take `const MessageReader&` as its parameter?